packages/graphql/schemas/cloud.graphql

```diff
@@ -0,0 +1,381 @@
+### This file was generated by Nexus Schema
+### Do not make changes to this file directly
+
+"""
+A CloudOrganization represents an Organization stored in the Cypress Cloud
+"""
+type CloudOrganization implements Node {
```

Comment:
Should we start using the Relay spec in the local schema too? Seems like the consistency would be valuable.

Reply:
Yep, working on pulling in helpers etc. along with writing the testing stub layer